packages/story-editor/src/components/canvas/displayElement.d.ts

```diff
@@ -13,9 +13,14 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-/**
- * External dependencies
- */
-import { createContext } from '@googleforcreators/react';
 
-export default createContext({ actions: {}, state: {} });
+import type { Element } from '@googleforcreators/elements';
+
+export interface DisplayElementProps {
+  element: Element;
+  previewMode: boolean;
```

Comment:
This one is optional too, but just not actively set to false if missing, just used as _falsy_ by being undefined. That should actually be fixed.